### DISCUSSION TITLE
Need DashboardChart.getTitle() that returns title of the chart.

### DISCUSSION THREAD

**Discussion by dewanprakash** [mongodb-js/charts-embed-sdk#101] [Ideas]
**Is your feature request related to a problem? Please describe.**
We are looking to implement an "Export data" at embedded dashboard level. Dashboard API has getAllChart API which returns an array of DashboardChart objects, which contain only the chartId (from the metadata perspective of the chart). We can download the data using getData(), however we don't get business context (i.e. title of chart etc) to establish the relation of specific data with specific chart. 

**Describe the solution you'd like**
`DashboardChart.getTitle()` that returns title of the chart.

**Describe alternatives you've considered**
We tried to create an object of Chart using the chartId we get from the `getAllChart()`, but we don't have the required details on Chart object as well. 

**Additional context**
We offer mongodb charts to our business users by embedding them in our solution. We have leveraged chart's SDK 3.3.1 for the same.

**Comment by khanguslee** [comment]
Hello @dewanprakash. I would recommend that you post this feature onto the [MongoDB Feedback Engine](https://feedback.mongodb.com/forums/923524-charts) so that we can prioritise this feature alongside other feature requests too. Thanks!

**Comment by khanguslee** [comment]
Hello @dewanprakash. There is somewhat of a 'workaround' but it's not quite useful but I thought I'd share it anyway.

You can call the `setRenderingSpecOverride` method with a title (https://github.com/mongodb-js/charts-embed-sdk/blob/master/docs/rendering-spec.md) and also call the `getRenderingSpecOverride` method. Unfortunately, the `getRenderingSpecOverride` method only retrieves whatever you have set in the rendering spec override option and not the original title.

You could argue that since you set it, you would know what the value is which is why this is not quite useful and a new idea is recommended.
